Automattic/woocommerce-payments
# Add empty payment method edit screen

Fixes #1507.

#### Changes proposed in this Pull Request





- Replace "Manage" and payment method name buttons with actual links to payment method settings pages in the *Payment methods* card.
- Add an empty payment method settings page with a placeholder usage example. The page includes:
  - Breadcrumbs.
  - Rewriting `$current_section` while rendering section list to mark "WooCommerce Payments" as active (based on [an implementation](https://github.com/Automattic/woocommerce-payments/pull/1580) by @frosso).





**Links to the payment method edit screen:**

![Markup on 2021-04-26 at 17:10:33](https://user-images.githubusercontent.com/1759681/116106864-c4bc5b00-a6b2-11eb-9ff1-5c56cad4e8f1.png)

**The payment method edit screen with placeholder content:**

![Markup on 2021-04-26 at 17:13:32](https://user-images.githubusercontent.com/1759681/116106869-c5ed8800-a6b2-11eb-9eec-e433c6427e90.png)

#### Testing instructions





1. Enable the grouped settings feature flag `_wcpay_feature_grouped_settings`.
1. Go to `WooCommerce > Settings > Payments > WooCommerce Payments`.
1. Click the "Manage" link (or method name) next to "giropay" to see the settings page.
1. Verify the breadcrumbs are displayed as expected and the "WooCommerce Payments" section is marked as active.

-------------------

  - Not added - hidden behind a feature flag.
